### DISCUSSION TITLE
Release v2023.2 - Wed 21/06/2023

### DISCUSSION THREAD

**Discussion by manuelaromero-guadaltel** [INSPIRE-MIF/helpdesk-validator#966] [Announcements]
Dear INSPIRE users,

The next release of the INSPIRE Reference Validator has been scheduled to take place next Wednesday 21/06/2023 from 07.30 to 15.30.
We will keep you informed with the release status and the [Production instance](https://inspire.ec.europa.eu/validator/home/index.html) availability during the process.
In the meantime, if you experience issues in the [Production instance](https://inspire.ec.europa.eu/validator/home/index.html), we kindly invite you to use the [Staging instance](http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/home/index.html) instead.

Please take into account that Test Reports will be deleted to deploy a fresh [Production instance](https://inspire.ec.europa.eu/validator/home/index.html), so we encourage you to save any Test Run that you may want to preserve.

We will keep you posted with the service status.

Kind regards,

**Comment by jenriquesoriano** [comment]
Dear Community, 

given the new set-up for the INSPIRE Reference Validator, we need to extend the testing period during the next hours in order to ensure it is working properly.
There is a stable version in the  [Production instance](https://inspire.ec.europa.eu/validator/home/index.html) but, while we conduct these latest tests, you might encounter occassional unavailabilities. 
As usual, if you experience issues, you may use the [Staging instance](http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/home/index.html) instead.
Please excuse the inconvenience.

Best regards.

**Comment by jenriquesoriano** [comment]
Dear INSPIRE users,

just inform that the v2023.2 release has been deployed successfully in the [production instance](https://inspire.ec.europa.eu/validator/home/index.html).
For further details, please visit the [release notes](https://github.com/INSPIRE-MIF/helpdesk-validator/releases/tag/v2023.2).

Kind regards,
